### PR TITLE
add image component

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -522,4 +522,23 @@
...
+
+.circle-image{
+    width: 20vw;
+    height: 20vw;
+    border-radius: 50%;
+    box-shadow: 5px 5px 10px var(--black);
+}
+
+.responsive-image{
+    width: 30vw;
```
Comment (RakshithBhat03) [inline components.css]: Using `width:100%` is a good practice for making the image responsive.

File: styles.css [hunk 2]
```diff
@@ -704,4 +704,29 @@ iframe{
 
 .horizontal-card .card-description{
     margin-left: 0.5rem;
+}
+
+.square-image{
```
Comment (RakshithBhat03) [inline styles.css]: You have repeated this class again.

File: styles.css [hunk 3]
```diff
@@ -704,4 +704,29 @@ iframe{
...
+    box-shadow: 5px 5px 10px var(--black);
+}
+
+.image-components-preview{
+    position: relative;
+    justify-content: center;
+    gap: 2rem;
+}
+
+.circle-image{
```
Comment (RakshithBhat03) [inline styles.css:722]: Same here as well.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a few changes, Everything else looks fine.

File: styles.css (hunk 3 above)
Comment (ganesh-kanchi) [reply styles.css:722]: made the changes thanks for the review

**Review by MonikaJhingan — APPROVED** [review]
When you make any webpage make sure that the background color is not too loud and sharp. You can implement this change in your upcoming projects.

**Comment by ganesh-kanchi** [comment]
> When you make any webpage make sure that the background color is not too loud and sharp. You can implement this change in your upcoming projects.

will keep that in mind. thanks for the suggestion.

**Pull request merged**